portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -42,14 +54,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = request.getParameter("text-input");
 
-    // Add comment to datastore as new entity.
-    // TODO: add id to identify comments by
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    // TODO: Add unique entity keys?
```

Comment:
We can remove the TODO if it works as-is.

Reply:
Sounds good! I will keep a note of it written down so that when I am thinking about extensions I can explore the benefits.